source/process/release-process.md

```diff
@@ -138,16 +136,21 @@ The final release is cut. If an urgent and important issue needs to be addressed
     - Post this checklist in Release channel
     - Verify all items in the last posted release checklist are complete, if not alert the release manager.
 2. Build:
-    - Tags a new release (e.g. 1.1.0) and runs an official build which should be essentially identical to the last RC
+    - Tags a new release (e.g. 1.1.0) and runs an official build which should be essentially identical to the last RC. 
 3. PM:
+    - Post in Release Discussion with links to the EE and Team Edition bits.
     - Submit GitLab ticket to take next Mattermost version in the Omnibus using [template](https://gitlab.com/gitlab-org/omnibus-gitlab/issues/1241)
     - Update the mattermost.org/download page
     - Add the download links to http://docs.mattermost.com/administration/upgrade.html#version-archive
     - Contact owners of [community installers](http://www.mattermost.org/installation/) or submit [PRs](https://github.com/tommyvn/mattermost-heroku/commit/94f7c5c0c5d7d2672fb6d62b6a560b4b5c1b5131) to update install version number.
-    - Close final GitHub RC meta ticket  
+    - Close GitHub meta ticket for the release.
+    - Merge the docs release branch to master and verify all changes on docs.mattermost.com once the build is up. Submit a correction PR for any incorrect formatting or other errors missed during the intitial review.
+    - Update [MVP page](https://www.mattermost.org/mvp/) with the most valuable contributor of the release.
+    - Submit [GitLab MR](https://gitlab.com/gitlab-org/omnibus-gitlab/merge_requests/871/diffs) to take next Mattermost version in the Omnibus.
+    - Update [Docker preview image to latest version](https://github.com/mattermost/mattermost-docker-preview/commit/1c34195d70b26bd4c82b5ef4fa0ebaf421096881).
 4. Marketing:
     - Finalize Mailchimp email blast
-    - Finalize blog post for mattermost.org set timer for 06:00 PDT on first non-holiday weekday morning after release
+    - Finalize blog post for mattermost.org set timer for 06:00 PDT on the day of release.
```

Comment:
Perhaps add `and` e.g.
`Finalize blog post for mattermost.org and set timer for 06:00 PDT on the day of release.`